.github/workflows/commercial-bundle-bump.yml

```diff
@@ -0,0 +1,107 @@
+name: Bump @guardian/commercial-bundle
+
+on:
+    schedule:
+        # runs every 5 minutes between 8am and 6pm, Monday to Friday
+        - cron: '*/5 08-18 * * 1-5'
```

Comment:
I picked every 5 minutes to have the best responsiveness to commercial releases!

The check stage of this workflow takes less than 5 seconds, that's why I feel it's not a ridiculous frequency, but I wouldn't mind reducing it a tad.

I'm not sure how it ties into the build time of frontend however?

Reply:
@mxdvl that was my inital idea too, but it requires adding and maintaining a PAT token which I didn't like the idea of https://github.com/guardian/commercial/pull/837/files

Reply:
@akash1810 did we ever create an org-wide PAT for this kind of thing?

Reply:
> @akash1810 did we ever create an org-wide PAT for this kind of thing?

We have not.

Reply:
> I picked every 5 minutes to have the best responsiveness to commercial releases!

Looking at https://github.com/guardian/commercial/releases suggests we are not releasing at a frequency of 5 minutes. The maximum seems to be twice a day.

> I'm not sure how it ties into the build time of frontend however?
 
When running every 5 minutes, I think at least half the runs are redundant as:

- 00:00:00 Workflow runs, and raises PR
- 00:00:01 TeamCity build starts
- 00:05:00 Workflow runs, and exits as PR is open ⬅️  this one feels redundant
- 00:10:00 TeamCity build completes
- 00:10:01 PR approved, and merged
- 00:15:00 Workflow runs, and raises PR

Reply:
The intension behind the frequent runs was to have the best responsiveness to commercial releases. So that one would only need to wait 5 minutes maximum for a PR to be opened after a release in commercial.

But I do concede that 5 minutes may be a little too frequent and have reduced to 15 mins :)